### PR TITLE
put ticker symbol back into title and change website urls to pepecoin…

### PR DESCRIPTION
**Pull request by KarzoGitHub**
- ticker symbol was put back into the website title

- changed website URLs to pepecoin.org

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,21 +4,21 @@
...
-  <meta name="description" content="Getpepecoin is the official info site for Pepecoin">
+  <title>Pepecoin (PEPE) - A Community Focused Cryptocurrency</title>
+  <meta name="title" content="Pepecoin (PEPE) - A Community Focused Cryptocurrency">
+  <meta name="description" content="Pepecoin.org is the official info site for Pepecoin">
 
   <meta property="og:type" content="website">
-  <meta property="og:url" content="https://getpepecoin.com/">
-  <meta property="og:title" content="GetPepecoin.com - A Community Focused Cryptocurrency">
+  <meta property="og:url" content="https://pepecoin.org/">
+  <meta property="og:title" content="Pepecoin.org - A Community Focused Cryptocurrency">
```
Comment (pepecoinppc) [inline src/index.html:13]: Can we change this to `Pepecoin (PEPE) - A Community Focused Cryptocurrency`?

File: src/index.html [hunk 2]
```diff
@@ -4,21 +4,21 @@
...
   <meta property="og:description" content="Pepecoin is a layer 1 cryptocurrency created by one of the original Dogecoin shibes from 2013. It was created for one purpose, to create a new and fun community just like the original Dogecoin community.">
-  <meta property="og:image" content="https://getpepecoin.com/assets/img/pepe_mainmeta_compressed.jpg">
+  <meta property="og:image" content="https://pepecoin.org/assets/img/pepe_mainmeta_compressed.jpg">
 
   <meta property="twitter:card" content="summary_large_image">
-  <meta property="twitter:url" content="https://getpepecoin.com">
-  <meta property="twitter:title" content="GetPepecoin.com - A Community Focused Cryptocurrency">
+  <meta property="twitter:url" content="https://pepecoin.org">
+  <meta property="twitter:title" content="Pepecoin.org - A Community Focused Cryptocurrency">
```
Comment (pepecoinppc) [inline src/index.html:19]: Can we change this to `Pepecoin (PEPE) - A Community Focused Cryptocurrency` as well?

File: src/index.html [hunk 3]
```diff
@@ -250,8 +250,8 @@ <h3 id="getting-started-title">Pepecoin Core</h3>
                   Send, receive, and manage your Pepecoin effortlessly with the intuitive wallet interface.
                 </p>
 
-                <a id="core-download-windows" href="" alt="Download button for Windows version of Pepecoin Core">
-                  <button type="button" class="btn btn-secondary btn-lg disabled" disabled>Windows</button>
+                <a id="core-download-windows" href="https://github.com/pepecoinppc/pepecoin/releases/tag/v1.0.0" alt="Download button for Windows version of Pepecoin Core">
```
Comment (pepecoinppc) [inline src/index.html:253]: Change to "Download button for Pepecoin Core"

File: src/index.html (hunk 1 above)
Comment (KarzoGitHub) [reply src/index.html:13]: sure

**Pull request merged**